README.md

```diff
@@ -23,16 +25,11 @@ npm install --save-dev raw-loader
 
 <h2 align="center">Usage</h2>
 
-**require**
-```js
-const txt = require("raw!./file.txt")
-```
+Use the loader either via your webpack config or manually.
 
-**webpack.config.js**
-```js
-const txt = require('file.txt')
-```
+### Via webpack config (recommended)

```

Comment:
if you want it to stay under the same (Usage) section normal markdown h3 `###` if it is on the 'top' level like Install, Usage, Options, [Example], Maintainer => `h2 align="center"></h2>`